app/Controllers/feedController.php

```diff
@@ -587,6 +588,11 @@ public static function actualizeFeeds(?int $feed_id = null, ?string $feed_url =
 								continue;
 							}
 
+							$entry->applyFilterActions($titlesAsRead);
```

Comment:
Regression: marking as read for duplicated titles should not apply to updated articles.
Fixed in https://github.com/FreshRSS/FreshRSS/pull/6664